source/HiringPanel.cpp

```diff
@@ -79,7 +77,7 @@ void HiringPanel::Draw()
 	info.SetString("passengers", to_string(passengers));
 
 	static const int DAILY_SALARY = 100;
-	int salary = DAILY_SALARY * (fleetRequired - 1);
+	int salary = DAILY_SALARY * (flagship ? fleetRequired - 1 : 0);
```

Comment:
Do we actually want to show a zero for the total crew salaries here just because there is no flagship right now?

Reply:
Well, it's better than showing -100...

It would show zero if they had one, too, if the crew requirement is one.

Reply:
I suppose the alternative is setting required crew to 1, and passenger space to -1 (passenger space can go negative in other contexts, so it's not that unusual).

Reply:
Would `fleetRequired - (flagship ? 1 : 0)` not be more appropriate, then? That is, only subtract 1 if we have a flagship.